pages/missions.php

```diff
@@ -294,6 +294,43 @@ function runActiveMission(): bool {
                     echo "You have been paid &yen;$mission->money.<br />";
                 }
 
+                // Village reputation
+                // TODO: Consider diminishing returns or daily/weekly caps
+                $reputation_gain = 1;
+                $reputation_chance = 50;
```

Comment:
Similar comments here, let's do fixed amounts of reputation instead of random chance